src/pudl/package_data/meta/xlsx_maps/eia861/column_maps/non_net_metering_eia861.csv

```diff
@@ -15,12 +15,12 @@ industrial_pv_capacity_mw,12,12,12
 transportation_pv_capacity_mw,13,13,13
 direct_connection_pv_capacity_mw,14,14,14
 total_pv_capacity_mw,15,15,15
-residential_storage_capacity_mw,16,16,16
-commercial_storage_capacity_mw,17,17,17
-industrial_storage_capacity_mw,18,18,18
-transportation_storage_capacity_mw,19,19,19
-direct_connection_storage_capacity_mw,20,20,20
-total_storage_capacity_mw,21,21,21
+residential_all_storage_capacity_mw,16,16,16
+commercial_all_storage_capacity_mw,17,17,17
+industrial_all_storage_capacity_mw,18,18,18
+transportation_all_storage_capacity_mw,19,19,19
+direct_connection_all_storage_capacity_mw,20,20,20
+total_all_storage_capacity_mw,21,21,21

```

Comment:
what does the all mean here?

Reply:
I had to find a way to differentiate between `pv_storage` and `storage` and because of the way the code reads these prefixes, so I added `all`

Reply:
ahh got it!